README.md

```diff
@@ -76,6 +76,72 @@ extension MyCustomViewController {
 }
 ```
 
+###Reusable Views To Improve Performance
+
+Collections and tables view use `reuseidentifier` on cell to recycle a view
+
+Each custom collection and table view controller is extended with this code and provide a list of available reuse identifier
+
+`Reusable` enumeration contains list of available reusable identifier
+
+`kind` property represent types of view (table, collection)
+
+`viewType` property return type of view if specified in storyboard.
+
+```swift
+extension MyCustomTableViewController { 
+
+    enum Reusable: String, Printable, ReusableProtocol {
+        case myCellId = "myCellId"
+        case mySecondCellId = "mySecondCellId"
+        
+        var kind: ReusableKind? {
+            ...
+        }
+         var viewType: UIView.Type? {
+            switch (self) {
+            case testreuse:
+                return TableViewCell.self
+            default:
+                return nil
+            }
+        }
+
+        var identifier: String? { return self.description } 
+        var description: String { return self.rawValue }
+    }
+}
+```
+
+example usage for dequeue a reusable view
+```swift
+class MyCustomTableViewController: UITableViewController, UITableViewDataSource {
+
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        return self.dequeueReusableCell(Reusable.myCellId, forIndexPath: indexPath) as! UITableViewCell
+    }
+    )
+```
+
+Before dequeue view you **must** register for each identifier a class or a xib
```

Comment:
it is not required by the Natalie right? it is up to programmer if he register it or not (like for Storyboard cells are automatically registered).

In general, leave example of usage of Natalie. No need to confuse about other things.